app-admin/logcheck/files/logcheck.cron

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+#
+# To enable sync via cron, execute "sudo -u logcheck touch /etc/logcheck/cron-logcheck-enabled"
+if [[ ! -f /etc/logcheck/cron-logcheck-enabled ]]; then
+   exit
+fi
+if [ ! -d /var/lock/logcheck ]; then
+	mkdir -p /var/lock/logcheck
+	chown logcheck:logcheck /var/lock/logcheck
+fi
+if [ -x /usr/sbin/logcheck ]; then
```

Comment:
Is this test really required? I would assume the binary always exists if logcheck.cron exists.